### PR TITLE
[DOC] Adds Iñigo's Community Bonding Period Blog

### PR DESCRIPTION
**Pull request by itellaetxe**
### Changes
- Added `/blogs/2024` directory for this year's GSoC blogs. Other GSoC mates can push their blogs to this folder.
- Added my community bonding period blog.

@deka27 @WassCodeur @robinroy03

### CODE REVIEW

File: posts/2024/2024_05_27_Inigo_week_0.rst
Comment (jhlegarreta) [inline posts/2024/2024_05_27_Inigo_week_0.rst:13]: I did not provide TractoInferno, right?

**Review by jhlegarreta — COMMENTED** [review]
Check the codespell error.

Looks good besides the inline comment.

Thanks.

File: posts/2024/2024_05_27_Inigo_week_0.rst
Comment (itellaetxe) [reply posts/2024/2024_05_27_Inigo_week_0.rst:13]: I also thought so, but if you check the [Zenodo entry for FINTA/CINTA/GESTA/FIESTA](https://zenodo.org/records/7562790) there is a file called `data_tractoinferno_hcp_qbx.hdf5`, so I guess it does come from TractoInferno. Am I wrong?

Comment (jhlegarreta) [reply posts/2024/2024_05_27_Inigo_week_0.rst:13]: OK, you're right. I was more meaning that for FiberCup I had to dig into my archives, but no need to add more detail. Resolving.

**Comment by deka27** [comment]
PR looks good to me, codespell issue happening due to previous blog post. I am not sure if we should correct it or not.

**Comment by skoudoro** [comment]
overall, it looks ok, but the first blog post is slightly different than the others

look at 
- https://fury.gl/latest/posts/2023/2023-05-29-week-0-joaodellagli.html
- https://dipy.org/posts/2023/2023_05_29_Shilpi_Week_0_1.html

And update with those inspiration

**Comment by itellaetxe** [comment]
@deka27 I also corrected the codespell errors, same as you did in your PR, so we don't have to rebase. Thanks for pointing that out.
@skoudoro I think it looks better now, more aligned with the inspo you suggested. Thanks

**Review by jhlegarreta — APPROVED** [review]

**Review by skoudoro — COMMENTED** [review]
Hi @itellaetxe , overall it looks good. See below for my comments.

Thank you for addressing them

File: posts/2024/2024_05_27_Inigo_week_0.rst
Comment (itellaetxe) [reply posts/2024/2024_05_27_Inigo_week_0.rst:13]: Done

**Comment by itellaetxe** [comment]
@skoudoro, I do not see any warnings now in the CI. I think it is ready to merge. I will go and post it in the Mastodon server once you merge it.

**Review by skoudoro — APPROVED** [review]
I was waiting for the review of @WassCodeur, but ok, I will go ahead and merge it. all good.

Thank you for this!

**Pull request merged**
